OpenRA.Mods.Common/Orders/RepairOrderGenerator.cs

```diff
@@ -49,17 +49,28 @@ static IEnumerable<Order> OrderInner(World world, MouseInput mi)
 			if (underCursor.Owner != world.LocalPlayer)
 				yield break;
 
+			Actor repairBuilding = null;
+			string orderId = "Repair";
+
 			// Test for generic Repairable (used on units).
 			var repairable = underCursor.TraitOrDefault<Repairable>();
-			if (repairable == null)
-				yield break;
+			if (repairable != null)
+				repairBuilding = repairable.FindRepairBuilding(underCursor);
+			else
+			{
+				// Test for generic RepairableNear (used on ships).
```

Comment:
There was already comment 'Test for generic Repairable', so I added this one for consistency. Should I remove only mine or both?